core/build.gradle

```diff
@@ -21,6 +21,11 @@ dependencies {
     compile "dk.brics.automaton:automaton:${AUTOMATON_VERSION}"
     compile "org.apache.commons:commons-csv:${COMMONS_CSV_VERSION}"
     implementation "com.github.javafaker:javafaker:${FAKER_VERSION}"
+    implementation ('org.yaml:snakeyaml') {
+        version {
+            strictly '1.26'
```

Comment:
This would be better as `strictly '[1.26,)'` so that updates to Faker that require a higher version of snakeyaml do not break the build

Reply:
Using require as:
> Implies that the selected version cannot be lower than what require accepts
but could be higher through conflict resolution, even if higher has an
exclusive higher bound.
>
> https://docs.gradle.org/current/userguide/rich_versions.html